akka-projection-testkit/src/main/scala/akka/projection/testkit/TestProjection.scala

```diff
@@ -4,6 +4,10 @@
 
 package akka.projection.testkit
```

Comment:
WIP. I used this PR to help build functionality of `TestProjection`, but it will be refined in #198.